dask_ml/model_selection/_incremental.py

```diff
@@ -853,17 +930,29 @@ def __init__(
         test_size=None,
         patience=False,
         tol=0.001,
-        scores_per_fit=1,
+        fits_per_score=1,
         max_iter=100,
         random_state=None,
         scoring=None,
+        verbose=False,
+        prefix="",
+        scores_per_fit=None,
     ):
+        if scores_per_fit is not None and fits_per_score != 1:
```

Comment:
Sorry, didn't realize this deprecation was in the `__init__`. This should be done in the `fit`. `__init__` isn't supposed to do any validation.

This will also fix the issue you saw with the warnings in the doc build.

Reply:
D'oh, of course. I've made that change, and removed the `:okwarning:`s. This PR should be ready merge now.